content/source/docs/enterprise/sentinel/integrate-vcs.html.md

```diff
@@ -1,10 +1,10 @@
 ---
 layout: enterprise2
-page_title: "Sentinel - Terraform Enterprise"
+page_title: "Integrating with VCS - Sentinel - Terraform Enterprise"
 sidebar_current: "docs-enterprise2-sentinel-integrate-vcs"
 ---
 
-# Integrate with VCS
+# Integrating with VCS
 
 Sentinel is designed to enable policy as code. As such, it is recommended that the policies are managed in a VCS service like Github, Bitbucket or Gitlab. Terraform Enterprise will add integration with these VCS providers to automatically ingress the policies from the VCS repository in an upcoming release. In the meantime it is recommended that the [Policy APIs](../api/policies.html) are used to automatically push changes from VCS to Terraform Enterprise using a CI/CD service like Jenkins, Drone, or Circle CI.
```

Comment:
As long as we're here, can we make the language on "in an upcoming release" softer? I'd prefer "in the future" if we have to mention this at all.

Reply:
TBH I don't want to touch this page at all right now, because I'm planning to replace the entire thing as soon as we release a version of the `tfe` provider that can handle policy sets, which might even be like next week. (Turns out it's pretty trivial to build a drop-in VCS integration by just throwing it all in a TFE workspace.)